### ISSUE TITLE
`await fetch` is dropped in 1.14.0

### ISSUE THREAD

**Issue by kenrick95** [rollup/rollup#2896]
- Rollup Version: 1.14.0
- Operating System (or Browser): Windows 10 1809
- Node Version: v10.15.3

### How Do We Reproduce?

[REPL](https://rollupjs.org/repl?version=1.14.0&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmltcG9ydCUyMGZldGNoQW5kRGVjb2RlJTIwZnJvbSUyMCcuJTJGdXRpbC5qcyclM0IlNUNuJTVDbmZldGNoQW5kRGVjb2RlKCklM0IlMjIlMkMlMjJpc0VudHJ5JTIyJTNBdHJ1ZSU3RCUyQyU3QiUyMm5hbWUlMjIlM0ElMjJ1dGlsLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmV4cG9ydCUyMGRlZmF1bHQlMjBhc3luYyUyMGZ1bmN0aW9uJTIwZmV0Y2hBbmREZWNvZGUoKSUyMCU3QiU1Q24lNUN0Y29uc3QlMjByZXNwb25zZSUyMCUzRCUyMGF3YWl0JTIwZmV0Y2goJ2h0dHBzJTNBJTJGJTJGZXhhbXBsZS5jb20nKSUzQiU1Q24lMjAlMjByZXR1cm4lMjByZXNwb25zZS5qc29uKCklM0IlNUNuJTdEJTIyJTdEJTVEJTJDJTIyb3B0aW9ucyUyMiUzQSU3QiUyMmZvcm1hdCUyMiUzQSUyMmNqcyUyMiUyQyUyMm5hbWUlMjIlM0ElMjJteUJ1bmRsZSUyMiUyQyUyMmFtZCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyJTIyJTdEJTJDJTIyZ2xvYmFscyUyMiUzQSU3QiU3RCU3RCUyQyUyMmV4YW1wbGUlMjIlM0FudWxsJTdE)

main.js

```js
import fetchAndDecode from './util.js';

fetchAndDecode();
```

util.js

```js
export default async function fetchAndDecode() {
	const response = await fetch('https://example.com');
	return response.json();
}
```

### Expected Behavior

`await` is not dropped. This is the UMD output from Rollup 1.13.0. [REPL](https://rollupjs.org/repl?version=1.13.0&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmltcG9ydCUyMGZldGNoQW5kRGVjb2RlJTIwZnJvbSUyMCcuJTJGdXRpbC5qcyclM0IlNUNuJTVDbmZldGNoQW5kRGVjb2RlKCklM0IlMjIlMkMlMjJpc0VudHJ5JTIyJTNBdHJ1ZSU3RCUyQyU3QiUyMm5hbWUlMjIlM0ElMjJ1dGlsLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmV4cG9ydCUyMGRlZmF1bHQlMjBhc3luYyUyMGZ1bmN0aW9uJTIwZmV0Y2hBbmREZWNvZGUoKSUyMCU3QiU1Q24lNUN0Y29uc3QlMjByZXNwb25zZSUyMCUzRCUyMGF3YWl0JTIwZmV0Y2goJ2h0dHBzJTNBJTJGJTJGZXhhbXBsZS5jb20nKSUzQiU1Q24lMjAlMjByZXR1cm4lMjByZXNwb25zZS5qc29uKCklM0IlNUNuJTdEJTIyJTdEJTVEJTJDJTIyb3B0aW9ucyUyMiUzQSU3QiUyMmZvcm1hdCUyMiUzQSUyMmNqcyUyMiUyQyUyMm5hbWUlMjIlM0ElMjJteUJ1bmRsZSUyMiUyQyUyMmFtZCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyJTIyJTdEJTJDJTIyZ2xvYmFscyUyMiUzQSU3QiU3RCU3RCUyQyUyMmV4YW1wbGUlMjIlM0FudWxsJTdE)

```js
(function (factory) {
	typeof define === 'function' && define.amd ? define(factory) :
	factory();
}(function () { 'use strict';

	async function fetchAndDecode() {
		const response = await fetch('https://example.com');
	 	return response.json();
	}

	fetchAndDecode();

}));

```

### Actual Behavior

This is the UMD output in Rollup 1.14.0

```js
(function (factory) {
	typeof define === 'function' && define.amd ? define(factory) :
	factory();
}(function () { 'use strict';

	async function fetchAndDecode() {
		const response; // <-- the `await` is dropped, creating a Syntax Error
	 	return response.json();
	}

	fetchAndDecode();

}));

```

**Comment by RomanHotsiy** [comment]
I'm seeing a similar more generic issue on `1.14.0`:

[REPL](https://rollupjs.org/repl?version=1.14.0&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmV4cG9ydCUyMGFzeW5jJTIwZnVuY3Rpb24lMjBoYW5kbGVyKCklMjAlN0IlNUNuJTVDdGF3YWl0JTIwdGVzdCgpJTNCJTVDbiU3RCU1Q24lNUNuYXN5bmMlMjBmdW5jdGlvbiUyMHRlc3QoKSUyMCU3QiU1Q24lNUN0Y29uc29sZS5sb2coJ3Rlc3QnKSUzQiU1Q24lN0QlMjIlMkMlMjJpc0VudHJ5JTIyJTNBdHJ1ZSU3RCU1RCUyQyUyMm9wdGlvbnMlMjIlM0ElN0IlMjJmb3JtYXQlMjIlM0ElMjJjanMlMjIlMkMlMjJuYW1lJTIyJTNBJTIybXlCdW5kbGUlMjIlMkMlMjJhbWQlMjIlM0ElN0IlMjJpZCUyMiUzQSUyMiUyMiU3RCUyQyUyMmdsb2JhbHMlMjIlM0ElN0IlN0QlN0QlMkMlMjJleGFtcGxlJTIyJTNBbnVsbCU3RA==)

main.js

```js
export async function handler() {
	await test();
}

async function test() {
	console.log('test');
}
```

## Expected Behavior
CJS output from rollup `1.14.0`:

```js
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

async function handler() {
	 await test();
}

async function test() {
	console.log('test');
}

exports.handler = handler;
```

## Actual Behavior
CJS output from rollup `1.14.0`:

```js
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

async function handler() {
	 await test();
}

exports.handler = handler;
```

The `test` function is dropped!

**Comment by lukastaegert** [comment]
Fix at #2899

**Comment by RomanHotsiy** [comment]
That was fast 🔥! Thanks!

**Comment by lukastaegert** [comment]
Sorry for the inconvenience, should be fixed in 1.14.1 now.

**Comment by kenrick95** [comment]
Thanks a lot!
